MycroftAI/mycroft-core
# Initial Spanish translation

## Description
This PR provides an initial translation of the text dialogs to Spanish

## How to test
If you start the CLI and you have `es-es` as the mycroft language (in `mycroft.conf`) you should see messages in Spanish, as well as errors due to missing translations in the default skills.

You will see: "Espera un momento mientras acabo de iniciarme por favor" (please wait until I finish booting)

As well as several errors such as `"IOError: [Errno 2] No such file or directory:  /opt/mycroft/skills/fallback-unknown/vocab/es-es/question.voc"`

Those skills still need to be translated

## Contributor license agreement signed?
CLA [x] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
